packages/realm_generator/test/good_test_data/private_fields.expected

```diff
@@ -13,15 +13,15 @@ class WithPrivateFields extends _WithPrivateFields
 
   WithPrivateFields(
     String _plain, {
-    int _withDefault = 0,
+    int withDefault = 0,
```

Comment:
This is actually fixing a bug in version 2.1.0. Named arguments cannot start with underscore.